packages/docs-reanimated/docs/debugging/logger-configuration.mdx

```diff
@@ -0,0 +1,67 @@
+---
+id: logger-configuration
+sidebar_label: Logger configuration
+sidebar_position: 2
+---
+
+# Logger configuration
+
+Reanimated shows warnings that indicate misuses of the library API, such as modifying the [shared value](/docs/fundamentals/glossary#shared-value) during component re-render. These logs can be configured to be more or less verbose. The **default** configuration shows **all warnings and errors**. You can change this behavior by using the `configureReanimatedLogger` function.
```

Comment:
the docs doesn't say whether the logger is enabled by default or the user needs to enable it themself

Reply:
Mhm, I thought that this part of the sentence `The **default** configuration shows **all warnings and errors**` is enough. This fragment explicitly tells that the user **can change** this behavior by using the `configureReanimatedLogger` function, but, when the default behavior is sufficient, no configuration must be made.

Also this part `Reanimated shows warnings that indicate misuses of the library API` tells that reanimated displays logs by default (so, at least to me, it means that the logger is enabled).

How do you want to make it more explicit? I don't think we have to describe internals in detail.